mrjob/fs/ssh.py

```diff
@@ -88,12 +90,17 @@ def _ssh_cmd_args(self, address, cmd_args):
                 key_pair_file = self._remote_key_pair_file
```

Comment:
I think we are never using this, right? (when i !=0)

Reply:
Oh, right, `self._remote_key_pair_file` is unused and should be removed.